### DISCUSSION TITLE
Prefixed IDs, but saved in the database without the prefix?

### DISCUSSION THREAD

**Discussion by atymic** [spatie/laravel-prefixed-ids#18] [General]
Seems like a bit of a waste to index the duplicate prefix on every record. 
Instead, we could add a cast to handle removal/addition of the prefix before writing to the db.

Thoughts?

**Comment by jeffreyvanhees** [comment]
This package enables us to retrieve the correct model using:

```php
Spatie\PrefixedIds\PrefixedIds::find('prefix_fekjlms');
```

So the prefixed ID must always be stored to identify the corresponding model.
